crates/teloxide-core/src/adaptors/parse_mode.rs

```diff
@@ -231,12 +231,13 @@ where
         close_forum_topic,
         reopen_forum_topic,
         delete_forum_topic,
+        unpin_all_forum_topic_messages,
```

Comment:
any particular reason why that one moved?

Reply:
Just want to be consistent across adaptors' `request_forward` invocations, not to break to the common sequence